app/players/player.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        pass
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass
+
+    def player_info(self) -> None:
```

Comment:
If the `player_info` method is intended to be implemented by all subclasses, consider marking it with `@abstractmethod` to enforce this requirement.